mail/astroid/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        astroidmail astroid 0.15 v
+
```

Comment:
Please add a revision field:
```
github.setup        astroidmail astroid 0.15 v
revison             0
```

The revision field is used to make changes to a port and signal to MacPorts infra to rebuild it even if the software's actual version number has not changed.